container/features/src/main/resources/features.xml

```diff
@@ -826,10 +826,19 @@
     <feature name="opennms-trapd-listener" description="OpenNMS :: Trapd :: Listener" version="${project.version}">
       <feature>camel-blueprint</feature>
       <feature>opennms-trapd</feature>
-
+       <bundle>mvn:org.opennms.features.minion/core-impl/${project.version}</bundle>
+	  <feature>scv</feature>
       <bundle>blueprint:mvn:org.opennms.features.events/org.opennms.features.events.traps/${project.version}/xml/blueprint-trapd-listener</bundle>
     </feature>
 
+	<feature name="scv" description="Secure Credentials Vault"
```

Comment:
But strange thing is minion-core,scv are not visible even after installing minion-core-api